src/components/home/PanelResult.vue

```diff
@@ -53,6 +130,17 @@
               </v-toolbar-title>
             </v-toolbar>
           </template>
+          <template v-slot:[`item.institution`]="{ item }">
+            <v-tooltip bottom>
+              <template v-slot:activator="{ on }">
+                <span class="pointer" v-if="Object.keys(item.institution).length > 0" @click.stop="openInstitutionDetails(item.institution)">
```

Comment:
use v-btn with href instead. Remove class .pointer

replace v-if="..." with a method instead to make it easier to debug

Reply:
updated

Reply:
Don't forget to change the span into a v-btn + remove pointer class. I was confused about the href, the @click does the trick.

Reply:
done